pkgs/tools/admin/adtool/default.nix

```diff
@@ -1,46 +1,67 @@
-{ lib, stdenv, fetchurl, openldap }:
+{ lib, stdenv, fetchFromGitHub, fetchurl, db, openldap }:
 
-stdenv.mkDerivation rec {
+let
+  inherit (lib) getDev getLib;
+
+  # adtools needs openldap 2.4.x
+  openldap' = openldap.overrideAttrs (old: rec {
+    version = "2.4.59";
```

Comment:
I'm not sure if that's a good idea: not only that you can't modify this openldap' any further from outside. It's also a two year old version that doesn't receive any security updates anymore.

Reply:
well I suppose if we go this route it should get meta.knownVulnerabilities

Reply:
> well I suppose if we go this route it should get meta.knownVulnerabilities

That's the least thing we should do, but I'm not convinced that we should revive dead software on unstable unless there's a very good reason.
Anyways, forwarded it to security discussion to get some more opinions.